### ISSUE TITLE
Always swipes left

### ISSUE THREAD

**Issue by Akrelion45** [inspectorG4dget/bumble-autoswipe#19]
I think i create a new issue for this. 

I got the script to run, but it just left swipes. i am not sure if the european height is the problem or my config. 

My config looks like this:
`dealbreakers:
familyPlansv2: 0.2

dealmakers:
drinkingv2: 0.9

throttle_ratio: 0

age:
  18: 1
  19: 1
  20: 1
  21: 1
  22: 1
  23: 1
  24: 1
  25: 1
  26: 1
  27: 1
  28: 1

cannabisv2:
  never: 0.5
  socially: 0.5
  frequently: 0.3

drinkingv2:
  never: 0.8
  sober: 0.8
  rarely: 0.8
  socially: 0.8
  frequently: 0.4

educationv2:
  high school: 0.8
  trade/tech school: 0.7
  in college: 0.6
  undergraduate degree: 0.5
  in grad school: 0.4
  graduate degree: 0.3

exercisev2:
  sometimes: 0.6
  active: 0.3
  almost never: 0.6
  never: 0.6

familyPlansv2:
  want someday: 1
  don�t want: 0.3
  have and want more: 0.1
  have and don�t want more: 0.1
  not sure yet: 0.6
  have kids: 0.1
  open to kids: 0.8

genderv2:
  woman: 1
  woman and nonbinary: 1
  cis woman: 1

heightv2:
  min: 5.0
  max: 7.0
  bufferMax: 8.0

intentionsv2:
  relationship: 1
  something casual: 0.5
  don�t know yet: 0.8
  marriage: 1

Politicsv2:
  apolitical: 1
  moderate: 1
  liberal: 0.8
  conservative: 1

religionv2:
  agnostic: 0.9
  christian: 1
  catholic: 0.8
  hindu: 0.6
  muslim: 0.1
  jewish: 0.1
  other: 0.7
  spiritual: 0.8
  atheist: 1
  buddhist: 0.5

smokingv2:
  socially: 0.6
  never: 0.8
  regularly: 0.4`

My log looks like this:

`2023-12-26 23:12:08,689 | Value "don’t know yet" not handled for attribute "intentionsv2"
2023-12-26 23:12:08,709 | Attribute "starSignv2" not handled. Found value "taurus"
2023-12-26 23:12:08,719 | Value "left" not handled for attribute "Politicsv2"
2023-12-26 23:12:08,752 | {"heightv2": "174 cm", "exercisev2": "active", "educationv2": "in college", "drinkingv2": "rarely", "smokingv2": "never", "intentionsv2": "don\u2019t know yet", "familyPlansv2": "not sure yet", "starSignv2": "taurus", "Politicsv2": "left", "bio": "w\u00e4re sweet, wenn mir jemand bass beibringen k\u00f6nnte :)", "prompts": {}, "throttle_ratio": 0.0} | 0.4987181504336566
2023-12-26 23:12:10,991 | Value "don’t know yet" not handled for attribute "intentionsv2"
2023-12-26 23:12:11,009 | Attribute "starSignv2" not handled. Found value "taurus"
2023-12-26 23:12:11,065 | {"heightv2": "169 cm", "exercisev2": "active", "drinkingv2": "socially", "smokingv2": "socially", "genderv2": "woman", "intentionsv2": "don\u2019t know yet", "familyPlansv2": "not sure yet", "starSignv2": "taurus", "bio": "insta: schreibercecilie", "prompts": {}, "throttle_ratio": 0.0} | 0.4987181504336566
2023-12-26 23:12:13,507 | Attribute "starSignv2" not handled. Found value "aries"
2023-12-26 23:12:13,540 | {"heightv2": "171 cm", "exercisev2": "sometimes", "drinkingv2": "socially", "genderv2": "woman", "intentionsv2": "relationship", "familyPlansv2": "want someday", "starSignv2": "aries", "bio": "ISFP-T", "prompts": {}, "throttle_ratio": 0.0} | 0.4987181504336566
`

Edit: Found out, it was indeed the height, after deleting the code block for height, since i am not carying for height anyway, it works. 

But it seems he doesnt like the special character for that:

Value "don’t know yet" not handled for attribute "intentionsv2"

i tried in the config file as: 
don\u2019t know yet:
or as
don’t know yet:
Which both didnt work

**Comment by inspectorG4dget** [comment]
Thank you for the bug report. I'm glad to see you find this useful.

`Don't know yet` should be copy/psate'd from the log. I did not know that was a valid value when I built this, so it needs to be added to the config. I'll push that in the next update.

The height issue is due to the fact that you have height in cm (169 cm, in the case of the example in your log), and the max height is set to `7.0`, which would correspond to 7 cm (I think you perhaps wanted it to mean 7m, which is not how it is interpreted).

I'll push a new update with these fixes soon
